### PR TITLE
PUB-2589 - Redesign select list types page

### PR DESCRIPTION
**Pull request by NatashaAlker**
### JIRA link

https://tools.hmcts.net/jira/browse/PUB-2589

### Change description

Redesign of the select list types page in the verified user subscription flow

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/service/SubscriptionService.ts [hunk 1]
```diff
@@ -493,11 +498,7 @@ export class SubscriptionService {
                 listType.jurisdictions.some(jurisdiction => courtJurisdictions.includes(jurisdiction)) &&
                 (listType.restrictedProvenances.length === 0 || listType.restrictedProvenances.includes(userRole))
             ) {
-                if (
-                    selectedListTypes == null ||
-                    selectedListTypes.length == 0 ||
-                    selectedListTypes.includes(listName)
-                ) {
+                if (selectedListTypes != null && selectedListTypes.includes(listName)) {
```
Comment (KianKwa) [inline src/main/service/SubscriptionService.ts:501]: Don't need the if/else statements here. The following lines can be simplified to:

listType.checked = selectedListTypes?.length && selectedListTypes.includes(listName);

Comment (NatashaAlker) [reply src/main/service/SubscriptionService.ts]: Changed :)

File: src/main/views/subscription-configure-list.njk [hunk 2]
```diff
@@ -14,78 +14,8 @@
...
-                text: filterClear,
-                href: '?clear=all'
-              },
-              categories: checkedFilters
-            },
-            optionsHtml: filterOptionsHtml
-          }) }}
-        </form>
-      </div>
       <div class="layout-width-three-fifths" id="content">
```
[the remark this answers is not in the document]
Comment (NatashaAlker) [reply src/main/views/subscription-configure-list.njk:19]: Weirdly it messes up the formatting when this div is removed
Screenshot 2024-10-09 at 14 21 57

[the remark this answers is not in the document]
Comment (KianKwa) [reply src/main/views/subscription-configure-list.njk:19]: That is so weird. The 'layout-width-three-fifth' class is doing opposite of what it should do. In this case, you can replace 'layout-width-three-fifth' with 'govuk-grid-column-full' that should work. The three-fifth class is a bit misleading there.

[the remark this answers is not in the document]
Comment (NatashaAlker) [reply src/main/views/subscription-configure-list.njk]: Changed :)

**Review by KianKwa — APPROVED** [review]

**Pull request closed without merging**